src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,102 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+
+    private static final int DEFAULT_CAPACITY = 10;
+    private Object[] elementData;
+    private int size = 0;
+
+    public ArrayList() {
+        elementData = new Object[DEFAULT_CAPACITY];
+    }
+
     @Override
     public void add(T value) {
-
+        lengthChange();
+        elementData[size] = value;
+        size = size + 1;
```

Comment:
```suggestion
        size++;
```
looks simpler, fix in other places